docs/snapshotting/snapshot-support.md

```diff
@@ -98,7 +98,9 @@ Firecracker is optimized for fast load/resume and it's designed to do some very
 sanity checks only on the vm state file. It only verifies integrity using a 64
 bit CRC value embedded in the vm state file, but this is only as a partial
 measure to protect against accidental corruption, as the disk files and memory
-file need to be secured as well.
+file need to be secured as well. It is important to note that snapshot
+deserialization and CRC computation are done at the same time. Hence,
+the latter can be validated only if the former is successful.
```

Comment:
I think this has changed. CRC is validated before deserialization.

Reply:
You are correct. I will modify this.